### PR TITLE
[10.0][MIG] sale_order_project

### PR DESCRIPTION
**Pull request by Abranes**
As I messed up with the rebase. This PR replaces #348

### CODE REVIEW

**Comment by Abranes** [comment]
@acsonefho your comment from #348 has been attended.

File: sale_order_project/models/sale.py [hunk 1]
```diff
@@ -1,64 +1,36 @@
...
-                limit=1)[:1])
-
-    related_project_id = fields.Many2one(
-        comodel_name='project.project', string='Project',
-        compute='_compute_related_project_id')
-
-    @api.model
-    def _prepare_project_vals(self, order):
+    def _prepare_project_vals(self):
         name = u" %s - %s - %s" % (
```
Comment (acsonefho) [inline sale_order_project/models/sale.py]: Please add a `self.ensure_one()` (every function are `@api.multi` by default)

**Review by acsonefho — CHANGES_REQUESTED** [review]

File: sale_order_project/models/sale.py (hunk 1 above)
Comment (Abranes) [reply sale_order_project/models/sale.py]: Done, I forgot in the migration.

**Review by fuentes73 — APPROVED** [review]

**Comment by fuentes73** [comment]
@acsonefho could you please approve the changes?

**Review by acsonefho — APPROVED** [review]

**Comment by acsonefho** [comment]
@fuentes010 It's done, LGTM (code only, not tested)

**Comment by fuentes73** [comment]
@acsonefho thanks! @pedrobaeza Can you merge this PR? What do you think?

**Pull request merged**